crawl-ref/docs/options_guide.txt

```diff
@@ -1619,6 +1620,29 @@ always_show_zot = false
         Whether to show the "Zot" status light at all times, even when you've
         got plenty of time left.
 
+consumables = ?!}
```

Comment:
I might prefer longer names for these that make clearer what they connect to, e.g. `consumables_panel`, `consumables_panel_filter`, etc.

Reply:
`show_unidentified_consumables` feels descriptive enough, but if it should be renamed to `consumables_panel_show_unidentified_items`, please  let me know.